pkg/validation/spec/info.go

```diff
@@ -133,14 +136,43 @@ func (v *VendorExtensible) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+func (v *VendorExtensible) UnmarshalYAML(value *yaml.Node) error {
+	// var d map[string]interface{}
+	if value.Kind != yaml.MappingNode {
+		return errors.New("expected mappingNode")
+	} else if len(value.Content)%2 != 0 {
+		return errors.New("expected even child nodes")
+	}
+
+	for i := 0; i < len(value.Content); i += 2 {
+		var keyStr string
+		if err := util.DecodeYAMLString(value.Content[i], &keyStr); err != nil {
+			return err
+		}
+
+		if strings.HasPrefix(keyStr, "x-") || strings.HasPrefix(keyStr, "X-") {
```

Comment:
Is the capitalization check capturing an edge case or do we already have places where `X-` is passed in?

Reply:
This extra check for "X-" is here only to mirror the UnmarshalJSON behavior. I think you wrote that ;)